<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="1" ignored="0" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="RegistrationCycle" duration-ms="11585" started-at="2019-03-20T23:17:51Z" finished-at="2019-03-20T23:18:03Z">
    <groups>
      <group name="A">
        <method signature="TC01_RegistrationCycleTest.userRegistration()[pri:0, instance:com.imbb.testCases.TC01_RegistrationCycleTest@55d56113]" name="userRegistration" class="com.imbb.testCases.TC01_RegistrationCycleTest"/>
        <method signature="TC01_RegistrationCycleTest.gmailUserActivate()[pri:0, instance:com.imbb.testCases.TC01_RegistrationCycleTest@55d56113]" name="gmailUserActivate" class="com.imbb.testCases.TC01_RegistrationCycleTest"/>
      </group> <!-- A -->
      <group name="B">
        <method signature="TC02_UploadLogoTest.uploadPicture()[pri:0, instance:com.imbb.testCases.TC02_UploadLogoTest@148080bb]" name="uploadPicture" class="com.imbb.testCases.TC02_UploadLogoTest"/>
      </group> <!-- B -->
    </groups>
    <test name="RegistrationCycleTest" duration-ms="11585" started-at="2019-03-20T23:17:51Z" finished-at="2019-03-20T23:18:03Z">
      <class name="com.imbb.testCases.TC02_UploadLogoTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.imbb.testCases.TC02_UploadLogoTest@148080bb]" name="setup" is-config="true" duration-ms="2701" started-at="2019-03-20T23:17:59Z" finished-at="2019-03-20T23:18:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="uploadPicture()[pri:0, instance:com.imbb.testCases.TC02_UploadLogoTest@148080bb]" name="uploadPicture" duration-ms="0" depends-on-groups="A" started-at="2019-03-20T23:18:02Z" finished-at="2019-03-20T23:18:02Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC02_UploadLogoTest.uploadPicture()[pri:0, instance:com.imbb.testCases.TC02_UploadLogoTest@148080bb] depends on not successfully finished methods in group "A"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC02_UploadLogoTest.uploadPicture()[pri:0, instance:com.imbb.testCases.TC02_UploadLogoTest@148080bb] depends on not successfully finished methods in group "A"
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- uploadPicture -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.imbb.testCases.TC02_UploadLogoTest@148080bb]" name="tearDown" is-config="true" duration-ms="770" started-at="2019-03-20T23:18:02Z" finished-at="2019-03-20T23:18:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.imbb.testCases.TC02_UploadLogoTest -->
      <class name="com.imbb.testCases.TC01_RegistrationCycleTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.imbb.testCases.TC01_RegistrationCycleTest@55d56113]" name="setup" is-config="true" duration-ms="4367" started-at="2019-03-20T23:17:51Z" finished-at="2019-03-20T23:17:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="userRegistration()[pri:0, instance:com.imbb.testCases.TC01_RegistrationCycleTest@55d56113]" name="userRegistration" duration-ms="2733" started-at="2019-03-20T23:17:56Z" finished-at="2019-03-20T23:17:59Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)
at com.imbb.testCases.TC01_RegistrationCycleTest.userRegistration(TC01_RegistrationCycleTest.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userRegistration -->
        <test-method status="SKIP" signature="gmailUserActivate()[pri:0, instance:com.imbb.testCases.TC01_RegistrationCycleTest@55d56113]" name="gmailUserActivate" duration-ms="0" started-at="2019-03-20T23:17:59Z" depends-on-methods="com.imbb.testCases.TC01_RegistrationCycleTest.userRegistration" finished-at="2019-03-20T23:17:59Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TC01_RegistrationCycleTest.gmailUserActivate()[pri:0, instance:com.imbb.testCases.TC01_RegistrationCycleTest@55d56113] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TC01_RegistrationCycleTest.gmailUserActivate()[pri:0, instance:com.imbb.testCases.TC01_RegistrationCycleTest@55d56113] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gmailUserActivate -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.imbb.testCases.TC01_RegistrationCycleTest@55d56113]" name="tearDown" is-config="true" duration-ms="681" started-at="2019-03-20T23:17:59Z" finished-at="2019-03-20T23:17:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.imbb.testCases.TC01_RegistrationCycleTest -->
    </test> <!-- RegistrationCycleTest -->
  </suite> <!-- RegistrationCycle -->
</testng-results>
